ebau_gwr/core/tests/ech_mock_data.py

```diff
@@ -1,6 +1,6 @@
 getConstructionProjectResponse = (
     b'<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\n'
-    b'<ns2:response xmlns="http://www.ech.ch/xmlns/eCH-0058/5" xmlns:ns2="http://www.ech.ch/xmlns/eCH-0216/2" xmlns:ns3="http://www.ech.ch/xmlns/eCH-0129/5" xmlns:ns4="http://www.ech.ch/xmlns/eCH-0044/4" xmlns:ns5="http://www.ech.ch/xmlns/eCH-0097/2" xmlns:ns6="http://www.ech.ch/xmlns/eCH-0010/6" xmlns:ns7="http://www.ech.ch/xmlns/eCH-0007/6" xmlns:ns8="http://www.ech.ch/xmlns/eCH-0008/3">\n'
+    b'<ns2:response xmlns="http://www.ech.ch/xmlns/eCH-0058/5" xmlns:ns2="http://www.ech.ch/xmlns/eCH-0216/1" xmlns:ns3="http://www.ech.ch/xmlns/eCH-0129/5" xmlns:ns4="http://www.ech.ch/xmlns/eCH-0044/4" xmlns:ns5="http://www.ech.ch/xmlns/eCH-0097/2" xmlns:ns6="http://www.ech.ch/xmlns/eCH-0010/6" xmlns:ns7="http://www.ech.ch/xmlns/eCH-0007/6" xmlns:ns8="http://www.ech.ch/xmlns/eCH-0008/3">\n'
```

Comment:
Hmm, it might be a better idea to put this into .xml files instead, and read those

Reply:
Done.